src/qml/pages/settings/SettingsProgressBar.qml

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2024 - PresentThe Bitcoin Core developers
```

Comment:
```suggestion
// Copyright (c) 2024-present The Bitcoin Core developers
```
nit
and add other files you created it has 2023 instead of 2024

Reply:
Great catch updated the headers with the latest push [ed27a5a](https://github.com/bitcoin-core/gui-qml/commit/ed27a5a523c4fc6586e758df940daa80e8d0dc41)